mit_ocw_debt_problem/CreditCard.java

```diff
@@ -0,0 +1,40 @@
+import static java.lang.System.out;
+import java.util.Scanner;
+
+public class CreditCard {
+
+    public static void main(String[] args) {
+	double balance, apr, monthRate, principal, totalPaid = 0.0; //Variables
+	
+	Scanner keyboard = new Scanner(System.in); //Input device
+	
+	//Get the requested values
+	out.print("Please enter the outstanding balance: $");
+	balance = keyboard.nextDouble();
+
+	out.print("Please enter the annual interest rate %: ");
+	apr = keyboard.nextDouble() / 100.0;
+	
+	out.print("Please enter the minimum monthly payment rate %: ");
+	monthRate = keyboard.nextDouble() / 100.0;
+	
+	//Go through each month and display relevant numbers
+	for(int i = 1; i < 13; i++) {
+	    out.println("Month " + i);
+	    totalPaid += (balance * monthRate);
+	    out.printf("Minimum monthly payment: $%.2f\n", (balance * monthRate));
+	    principal = balance * (apr / 12.0);
+	    out.printf("Principal paid: $%.2f\n", principal);
+	    balance -= principal;
+	    out.printf("Remaining balance: $%.2f\n\n", balance);
+	}
+	
+	//Display final results
+	out.println("RESULT");
+	out.printf("Total amount paid: $%.2f\n", totalPaid);
+	out.printf("Remaining balance :$%.2f\n", balance);
```

Comment:
There's a little hard-to-notice detail in the description of problem 1:
When the value has no cents we show one decimal place, otherwise show 2 decimal place
eg:`'$320.0'` and `'$320.10'`

Does this solution cover for this case?

Feel free to submit a follow-up pull request if you feel like extending on the solution or refactoring!